locales/en.json

```
@@ -28,6 +29,8 @@
   "config.annotation_in_place": "Show annotation in place instead of appending to the tail",
   "config.annotation_max_length": "Max number of characters shown in the inline annotations. Excess characters will display as an ellipses (...)",
   "config.annotations": "Enable the inline annotations.",
+  "config.deelLog": "Show DeepL engine debug logs",
+  "config.deepLApiKey": "API key to use DeepL translate engine",
```

```suggestion
  "config.deepl_log": "Show DeepL engine debug logs",
  "config.deepl_api_key": "API key to use DeepL translate engine",
```

Please use the "Rename Key" function to update all the locales